src/iaas-utils/ha/gcp/clients/GCPServiceClient.go

```diff
@@ -0,0 +1,76 @@
+package clients
+
+import (
+	"iaas-utils/ha/common/models"
+	"iaas-utils/ha/common/interfaces"
+	"iaas-utils/ha/common/utils/apiutils"
+	"iaas-utils/ha/gcp/services"
+	"log"
+)
+
+type GCPServiceClient struct {
+	iaasDescriptors      models.IaaSDescriptors
+	authorizationRequest models.AuthorizationRequest
+	authorizationToken   *models.AuthorizationToken
+	authorizationService interfaces.IAuthorizationService
+	provisioningWaitTime float64
+	provisioningPollTime float64
+}
+
+func (svc *GCPServiceClient) Initialize(params ...interface{}) int {
+
```

Comment:
are these magic numbers ?

Reply:
yes.. seems like that.. I will move them to constants